style.css

```diff
@@ -0,0 +1,105 @@
+@import url("https://fonts.googleapis.com/css2?family=Bebas+Neue&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Nunito+Sans:wght@200&display=swap");
```

Comment:
:red_circle: Kindly make sure you import the right google font to match the Figma Design. The figma template uses `font-family: 'Poppins';`  

|Your Design|Figma Design|
|:--------------:|:------------------:|
|![family](https://user-images.githubusercontent.com/26736582/212389151-9bf4bf2d-1bca-4512-9373-8ea6da53482e.png)|![font-family](https://user-images.githubusercontent.com/26736582/212389191-43d13324-be94-43de-a63b-4ded6af1415b.png)|